Dockerfiles/manifests/cluster-agent/dca-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: datadog-auth-token
+type: Opaque
+data:
+  # use a base64 string: `echo -n <Thirty 2 characters long token> | base64`
+  token: PFRoaXJ0eSAyIGNoYXJhY3RlcnMgbG9uZyB0b2tlbj4=
```

Comment:
r/`<Thirty 2 characters long token>`/`<32_CHARACTERS_LONG_TOKEN>`

Should we not enforce a default token? to avoid any security issue..

Reply:
to be discussed